testdata/src/a/noprintflike/example.go

```diff
@@ -23,15 +23,15 @@ func ExamplePrintfLike() {
 	log.Info("%[3]*.[2*[1]f", "intKey", 1)
 
 	// Check message and key value pairs
-	log.Info("message", "key%s", "value %d") // want `logging message should not use format specifier "%s"`
+	log.Info("message", "key%s", "value %d") // want `logging message should not contain format specifiers, found "%s"`
 
-	log.Info("%[3]*s x") // want `logging message should not use format specifier ".+"`
-	log.Info("%[3]d x")  // want `logging message should not use format specifier ".+"`
+	log.Info("%[3]*s x") // want `logging message should not contain format specifiers, found ".+"`
```

Comment:
My comment is not on what you changed, but on what was reported here

`%[3]s` leads to `.+` ?

It's something that come from the external lib code imported from golang.org/x/tool but it's unclear to me what it could mean.

Do you?